### PR TITLE
Update systemd to 249.11-0ubuntu3.6

### PR DESCRIPTION
**Pull request by isantop**
Updates to the latest systemd from upstream Ubuntu. 

Fixes pop-os/raspi-img#16

### CODE REVIEW

**Review by mmstick — DISMISSED** [review]

**Review by jacobgkau — CHANGES_REQUESTED** [review]
Steam is not installing with this branch:

```
system76@pop-os:~$ sudo apt install steam
[sudo] password for system76: 
Reading package lists... Done
Building dependency tree... Done
Reading state information... Done
Some packages could not be installed. This may mean that you have
requested an impossible situation or if you are using the unstable
distribution that some required packages have not yet been created
or been moved out of Incoming.
The following information may help to resolve the situation:

The following packages have unmet dependencies:
 steam:i386 : Depends: libudev1:i386 but it is not going to be installed
              Recommends: libasound2-plugins:i386 but it is not installable
              Recommends: libsdl2-2.0-0:i386 but it is not installable
              Recommends: pipewire:i386 but it is not installable
E: Unable to correct problems, you have held broken packages.
```

**Comment by isantop** [comment]
It appears this also breaks sound on Oryp6. Reverting to the previous version also doesn't fix it, so that will definitely need to be fixed first.

File: debian/control [hunk 1]
```diff
@@ -442,3 +452,15 @@ Description: Userspace out-of-memory (OOM) killer
  systemd-oomd is a system service that uses cgroups-v2 and
  pressure stall information (PSI) to monitor and take action on
  processes before an OOM occurs in kernel space.
+
+Package: systemd-repart
+Section: admin
+Architecture: linux-any
```
Comment (jackpot51) [inline debian/control]: Here you should add i386, or just replace with amd64 arm64 i386 like above

**Comment by jackpot51** [comment]
Rebased on instead of merging upstream

**Comment by jacobgkau** [comment]
Still seem to be missing i386 builds on this branch (`libudev1:i386` doesn't exist, so can't install Steam.)

**Comment by jacobgkau** [comment]
Still missing i386 builds.

**Comment by 13r0ck** [comment]
@jacobgkau 1352e5c28b91fb7ebe3963c00b7375884e9cdf4d should help. but it looks like the build server is still down...

File: debian/control (hunk 1 above)
Comment (13r0ck) [reply debian/control]: https://github.com/pop-os/systemd/pull/26#issuecomment-1367648109

**Review by jacobgkau — APPROVED** [review]
The issue installing Steam is fixed with the latest commit; 1352e5c has `libudev1:i386` available.

Installing this update pulls in a new package, `systemd-hwe-hwdb`. This doesn't cause any problems with a normal `apt upgrade`.

Regression testing passed:
- [X] Confirm all Pop!_OS patches are still present
- [X] Shutdown/reboot work
- [X] Able to use the boot menu & updating initramfs works normally (systemd-boot)
- [X] Service management works normally
- [X] Virtual machine works
- [X] Suspend/resume works (in NVIDIA mode and Integrated mode)
- [X] DNS resolution works w/ automatic and custom settings (systemd-resolved)
- [X] Disabling/enabling chrony NTP works (timedatectl)
- [X] Able to view logs with journalctl (systemd-journald)
- [X] Check for Steam packaging issues
  - Make sure Steam can be installed with new systemd
  - Make sure systemd can be updated with Steam installed

**Review by 13r0ck — APPROVED** [review]

**Pull request merged**